core-api/package.json

```diff
@@ -5,7 +5,8 @@
   "license": "MIT",
   "scripts": {
     "start": "ts-node ./src/app.ts",
-    "start:dev": "ts-node-dev ./src/app.ts"
+    "start:dev": "ts-node-dev ./src/app.ts",
+    "typeorm": "ts-node ./node_modules/.bin/typeorm"
```

Comment:
@eduardosm7 Line 9 is the only one that matters here, it tells npm to run typeorm with ts-node, making it possible to execute .ts files without generating the .js. It's easier to work with migrations like this because typeorm generates .ts files.